package.json

```diff
@@ -74,7 +74,8 @@
     "save-latest-run": "./lighthouse-core/scripts/save-latest-run.sh",
     "compile-proto": "protoc --python_out=./ ./proto/lighthouse-result.proto && mv ./proto/*_pb2.py ./proto/scripts || (echo \"❌ Install protobuf ≥ 3.7.1 to compile the proto file.\" && false)",
     "build-proto-roundtrip": "mkdir -p .tmp && cross-env PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION_VERSION=2 python proto/scripts/json_roundtrip_via_proto.py",
-    "static-server": "node lighthouse-cli/test/fixtures/static-server.js"
+    "static-server": "node lighthouse-cli/test/fixtures/static-server.js",
+    "serve-gh-pages": "cd dist/gh-pages && python -m SimpleHTTPServer"
```

Comment:
when should we be using `serve-gh-pages` vs the static-server?

Reply:
static-server only does viewer b/c of a hard coded approval. Only for puppeteer test.

use serve-gh-pages if you want local viewer with a reasonable path (not prefixed with dist/) (or any app in gp-pages/)

Reply:
ah gotcha, throw this line into viewer's README then? (also can update the `dist/viewer` path reference while you're there 😃 )